### PR TITLE
Add Diners Club Numbers support

### PR DESCRIPTION
**Pull request by kurotaky**
Hi, I added Diners Club Numbers support.
for example `3056 930902 5904`, `3852 000002 3237`, and so on.
following numbers below.
https://www.discovernetworkvar.com/common/pdf/var/9-2_VAR_ALERT_Sep_2009.pdf

### CODE REVIEW

**Comment by kurotaky** [comment]
@wangjohn 
I added Diners Club support.
please review this pull request. :smiley:

File: src/creditly.js [hunk 1]
```diff
@@ -110,11 +127,15 @@ var Creditly = (function() {
...
         var isAmericanExpressCard = isAmericanExpress(number);
-        var maximumLength = getMaximumLength(isAmericanExpressCard);
+        var isDinersClubCard = isDinersClub(number);
+        var cardBrandName = cardBrand(number);
+        var maximumLength = getMaximumLength(cardBrandName);
         if (shouldProcessInput(e, maximumLength, selector)) {
           var newInput;
           if (isAmericanExpressCard) {
             newInput = addSpaces(number, americanExpressSpaces);
+          } else if (isDinersClubCard) {
```
Comment (wangjohn) [inline src/creditly.js]: Could we use the `cardBrandName` variable that we've already computed?

Comment (kurotaky) [inline src/creditly.js]: @wangjohn 
OK! I just fixed it. this like https://github.com/kurotaky/creditly/commit/1a70c243adeaf4d057449bfe61b53bbe371a621d.

**Comment by kurotaky** [comment]
@wangjohn 
Please review this commit (https://github.com/kurotaky/creditly/commit/1a70c243adeaf4d057449bfe61b53bbe371a621d) if you have time :smile:

**Comment by kurotaky** [comment]
How's that? : )

**Pull request closed without merging**